.github/workflows/test_build.yml

```diff
@@ -0,0 +1,59 @@
+name: Test Build
+
+on:
+  push:
+
+jobs:
+  test:
+    name: PHP 7.3, Laravel 6.*
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: 7.3
+
+      - name: Cache Composer dependencies
+        uses: actions/cache@v1
+        with:
+          path: ~/.composer/cache/files
+          key: php-7.3-laravel-6.*-composer-${{ hashFiles('**/composer.lock') }}
+          restore-keys: php-7.3-laravel-6.*-composer-
+
+      - name: Cache npm dependencies
+        uses: actions/cache@v1
+        with:
+          path: ~/.npm
+          key: npm-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: npm-
+
+      - name: Copy ENV Laravel Configuration for CI
+        run: php -r "file_exists('.env') || copy('.env.example', '.env');"
+
+      - name: Install dependencies
+        run: |
+          composer require "orchestra/testbench:4.*" --no-interaction --no-update
+          composer update --prefer-dist --no-interaction --no-progress --no-suggest
```

Comment:
Do we want to `composer update` before running our tests? I would assume we'd want to use the existing `composer.lock`?

Reply:
Yes, good catch!